### PR TITLE
Add order creation to the enterprise customer manual enroll tool

### PR DESCRIPTION
**Pull request by Tj-Tracy**
**Merge checklist:**
- [ ] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
- [ ] `make static` has been run to update webpack bundling if any static content was updated
- [X] [Version](https://github.com/edx/edx-enterprise/blob/master/enterprise/__init__.py) bumped
- [X] [Changelog](https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst) record added
- [ ] Translations updated

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/edx/edx-enterprise/releases) released
- [ ] Version pushed to [PyPI](https://pypi.org/project/edx-enterprise/)
- [ ] PR created in [edx-platform](https://github.com/edx/edx-platform) to upgrade dependencies (including edx-enterprise)

### CODE REVIEW

File: enterprise/admin/views.py [hunk 1]
```diff
@@ -791,7 +792,15 @@ def _enroll_users(
...
             if pending:
                 pending_messages.append(cls.get_pending_enrollment_message(pending, program_identifier))
-
+        
+        enrollments = map(lambda success: {
+            "lms_user_id": success.id,
+            "email": success.email,
+            "username": success.username,
+            "course_run_key": course_id,
+        }, succeeded)
```
Comment (tuchfarber) [inline enterprise/admin/views.py]: While `map`s work and make sense from a functional point of view, a list comprehension here is a little more in line with the rest of our code base and they're generally more "pythonic". 

Fun fact: `map`, `reduce` and `filter` were all actually going to be removed from Python 3, but some people got real mad about that, so they stayed.

File: enterprise/admin/views.py [hunk 2]
```diff
@@ -748,7 +749,7 @@ def _enroll_users(
             notify: Whether to notify (by email) the users that have been enrolled
         """
         pending_messages = []
-
+        ecommerceApi = EcommerceApiClient(request.user);
```
[the remark this answers is not in the document]
Comment (Tj-Tracy) [reply enterprise/admin/views.py]: Interesting - I was going to ask this question earlier today, but decided "I'll just write it like a JS developer and someone will correct me." 

I'll refactor it - good learning experience 😃

File: enterprise/admin/views.py (hunk 1 above)
Comment (Tj-Tracy) [reply enterprise/admin/views.py]: Interesting - I was going to ask this question earlier today, but decided "I'll just write it like a JS developer and someone will correct me."

I'll refactor it - good learning experience 😃

File: CHANGELOG.rst [hunk 3]
```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 ----------
 
+[2.0.11] - 2019-11-1
+---------------------
+* Added a Ecommerce API client call to Enterprise Customer manual enrollment form
```
Comment (crice100) [inline CHANGELOG.rst]: Don't forget to bump the version

File: enterprise/admin/views.py (hunk 1 above)
Comment (Tj-Tracy) [reply enterprise/admin/views.py]: Refactored!

File: CHANGELOG.rst (hunk 3 above)
Comment (Tj-Tracy) [reply CHANGELOG.rst]: version bumped.

**Pull request merged**